src/ip/Intercept.cc

```diff
@@ -13,6 +13,8 @@
 
 #include "squid.h"
 #include "comm/Connection.h"
+#include "compat/socket.h"
```

Comment:
With this you can remove the redundant `sys/socket.h` include below.

Reply:
removed